tests/Integration/QueryDataTableTest.php

```diff
@@ -26,19 +26,37 @@ public function it_can_set_total_records()
         $crawler->assertJson([
             'draw' => 0,
             'recordsTotal' => 10,
-            'recordsFiltered' => 20,
+            'recordsFiltered' => 10,
         ]);
     }
 
     #[Test]
     public function it_can_set_zero_total_records()
     {
         $crawler = $this->call('GET', '/zero-total-records');
+        $crawler->assertJson([
+            'draw' => 0,
+            'recordsTotal' => 0,
+            'recordsFiltered' => 0,
+        ]);
+    }
+
+    #[Test]
+    public function it_can_set_skip_total_records()
+    {
+        DB::enableQueryLog();
+
+        $crawler = $this->call('GET', '/skip-total-records');
         $crawler->assertJson([
             'draw' => 0,
             'recordsTotal' => 0,
```

Comment:
I think this is where the issue in the pagination/info occurs. Maybe we can set the recordsTotal to equal to recordsFiltered when we use the skipTotalRecords function?

Great test and I like the idea of asserting the query count.

Thanks!

Reply:
Sure! This way it will not say `(filtered from 0 total entries)`. I have attached some screenshots of the before/after.

<details>
<summary>Before</summary>

No filter:

![image](https://github.com/user-attachments/assets/85b318b2-7b8d-4817-abd1-089c4b4bf118)

With filter:

![image](https://github.com/user-attachments/assets/a4db4d61-fc77-4058-82b9-2f1753718277)

</details>

<details>
<summary>After</summary>

No filter:

![image](https://github.com/user-attachments/assets/1ba26813-58d0-4f7a-ad41-31c4d53a0006)


With filter:

![image](https://github.com/user-attachments/assets/a2c564f9-6307-41f4-9985-09e8fdfea452)


</details>